spec/API_specification/array_object.md

```diff
@@ -458,6 +458,8 @@ Exports the array for consumption by {ref}`function-from_dlpack` as a DLPack cap
 
         For other device types which do have a stream, queue or similar synchronization mechanism, the most appropriate type to use for `stream` is not yet determined. E.g., for SYCL one may want to use an object containing an in-order `cl::sycl::queue`. This is allowed when libraries agree on such a convention, and may be standardized in a future version of this API standard.
 
+        Support for any `stream` value other than `None` is optional and up to each library in question.
+
```

Comment:
```suggestion
        ```{note}
        Support for a `stream` value other than `None` is optional and implementation-dependent.
        ```
```

```